### PR TITLE
Rename caseInsensitive to caseSensitive

### PR DESCRIPTION
**Pull request by lukasmalkmus**

### CODE REVIEW

**Review by bahlo — APPROVED** [review]

**Comment by lukasmalkmus** [comment]
Let me fix the commit name. Something went wrong here.

File: axiom/monitors_test.go [hunk 1]
```diff
@@ -298,7 +298,7 @@ func TestMonitor_MarshalJSON(t *testing.T) {
 				"op": "",
 				"field": "",
 				"value": null,
-				"caseInsensitive": false,
+				"caseSensitive": false,
```
Comment (bahlo) [inline axiom/monitors_test.go:301]:
```suggestion
				"caseSensitive": true,
```
Shouldn't this be flipped?

Comment (lukasmalkmus) [reply axiom/monitors_test.go:301]: No. The test payload which gets encoded does not explicitly specify `monitor.CaseSensitive = true`. The zero value of `bool` is still false.

Comment (bahlo) [reply axiom/monitors_test.go:301]: Didn't look at the test, sounds good 👍

**Review by bahlo — APPROVED** [review]

**Pull request merged**